packages/now-cli/src/commands/deploy/latest.js

```diff
@@ -162,7 +162,11 @@ const printDeploymentStatus = async (
         prependEmoji(
           `${chalk.dim(indication.payload)}`,
           emoji(indication.type)
-        ) + `\n`
+        ) +
+          `\n` +
+          indication.info
+          ? `Learn More: ${indication.info}`
```

Comment:
For errors, we use two properties:
- `action: 'Learn More'`
- `link: https://vercel.link/foo`

Can we use `action` and `link` for warnings too?